tests/data/SLC6A4_active_excape_export.csv

```diff
@@ -0,0 +1,7229 @@
+Ambit_InchiKey,Original_Entry_ID,Entrez_ID,Activity_Flag,pXC50,DB,Original_Assay_ID,Tax_ID,Gene_Symbol,Ortholog_Group,SMILES
```

Comment:
CSV is 1MB I thought it was small enough to be added but I could trim it to just a few rows to keep at small as possible for debugging

Reply:
1 MB is not that much, but I do try to keep my git repo's lean. It could be good if we could still build some models with a decent performance for basic testing, so it's a bit of a tradeoff there.